raiden/transfer/token_network.py

```diff
@@ -29,7 +29,10 @@
     ContractReceiveUpdateTransfer,
 ]
 
-
+# State changes that should be duplicated in case there are two light clients
+DuplicableStateChanges = [
```

Comment:
I think we can rename this to represent that "we are dispatching to multiple channels because the channel is related with two light clients of the same hub"